pkg/webhooks/webhooks.go

```diff
@@ -60,4 +61,5 @@ func NewCRDValidationWebhook(ctx context.Context, _ configmap.Watcher) *controll
 var Resources = map[schema.GroupVersionKind]resourcesemantics.GenericCRD{
 	v1alpha1.SchemeGroupVersion.WithKind("AWSNodeTemplate"): &v1alpha1.AWSNodeTemplate{},
 	corev1alpha5.SchemeGroupVersion.WithKind("Provisioner"): &v1alpha5.Provisioner{},
+	v1beta1.SchemeGroupVersion.WithKind("EC2NodeClass"):     &v1beta1.EC2NodeClass{},
```

Comment:
No NodePool needed here?

Reply:
Nope! Those webhook validations were added in the core operator package